entities/storobj/storage_object.go

```diff
@@ -565,6 +562,17 @@ func (ko *Object) asVectors(in map[string][]float32) models.Vectors {
 	return nil
 }
 
+func (ko *Object) asMultiVectors(in map[string][][]float32) models.MultiVectors {
```

Comment:
**question:**  can't we change the type of `ko.MultiVectors` from `map[string][][]float32` to `models.MultiVectors` , or this will come in next PRs ?

Reply:
it's the same situation as with `Vectors` and this translation is done only once, here where we work with search results. I will think about this in the following PR's as I will be touching this implementation once again